### PR TITLE
Language extractor

### PR DESCRIPTION
**Pull request by redcarti**

### CODE REVIEW

File: lang.js [hunk 1]
```diff
@@ -1,62 +1,32 @@
...
+const LauncherDownload = require('minecraft-wrap').LauncherDownload
 const path = require('path')
-const getMinecraftFiles = require('./get_minecraft_files')
-
-if (process.argv.length !== 5) {
-  console.log('Usage : node lang.js <version1,version2,...> <output_dir> <temporary_dir>')
-  process.exit(1)
+const fs = require('fs')
+if (!(process.argv.length >= 3 && process.argv.length <= 4)) {
+  console.log('Usage : node lang.js <version>')
```
Comment (rom1504) [inline lang.js:5]: can you make the output folder configurable ?

Comment (redcarti) [reply lang.js:5]: yea

Comment (rom1504) [reply lang.js]: and being able to specify several languages would also be quite convenient

Comment (rom1504) [reply lang.js]: and versions

Comment (redcarti) [reply lang.js]: okay well

**Comment by redcarti** [comment]
there's the bug with multiple versions, it brokes with `TypeError: Cannot destructure property `hash` of 'undefined' or 'null'.`

**Comment by redcarti** [comment]
nvm, fixed

**Comment by rom1504** [comment]
thanks, could you use this to extract the languages and put them in https://github.com/PrismarineJS/minecraft-data ?

**Comment by redcarti** [comment]
> thanks, could you use this to extract the languages and put them in https://github.com/PrismarineJS/minecraft-data ?

yeah, will do it

**Pull request closed without merging**